modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/edit_user_navigation.jsp

```diff
@@ -64,8 +64,6 @@ redirect = HttpUtil.addParameter(redirect, liferayPortletResponse.getNamespace()
 redirect = HttpUtil.addParameter(redirect, liferayPortletResponse.getNamespace() + "screenNavigationEntryKey", screenNavigationEntryKey);
 %>
 
-<liferay-ui:success key="userAdded" message="the-user-was-created-successfully" />
```

Comment:
@sontruongces should not remove this one. There are some cases it redirects to this page and we need this information.